inst/sql/sql_server/concept_plausible_unit_concept_ids.sql

```diff
@@ -54,6 +57,5 @@ FROM
     		AND c.cohort_definition_id = @cohortDefinitionId
   	}
 	WHERE m.@cdmFieldName = @conceptId
-	  	AND value_as_number IS NOT NULL
```

Comment:
AND unit_concept_id != 0 to exclude 0 from denominator (which are checked in standardConceptRecordCompleteness)

Reply:
I think you made a very good argument for the change :-)